osbs/build/build_request.py

```diff
@@ -37,6 +38,15 @@ def register_build_class(cls):
     return cls
 
 
+def ditch_http_prefix(val):
+    if not val:
+        return val
+    # We don't want the scheme
+    return re.sub(r'^https?://(.*)$',
+                  lambda m: m.groups()[0],
+                  val)
```

Comment:
What about moving this code into the `RegistryURI` class? It would have three attributes - `uri`, `version` and something like `hostport` that would be used in places where `ditch_http_prefix(...)` is now.